### DISCUSSION TITLE
FR: Need a Markdown Rendering Widget Like `pager`

### DISCUSSION THREAD

**Discussion by Klaymr** [charmbracelet/huh#552] [Ideas]
The Note is not enough, and can't support pagination or longer context. Needs something like `gum pager`

**Comment by bashbunni** [comment]
Hey @Klaymr, thank you so much for wanting to make this project better. I've converted this to a discussion so we can talk through it and keep our backlog clear. If it does end up being something we want to add, we'll promote it back to an issue when we are ready to work on it.

If we commit to this feature, that's a feature we will need to maintain for the lifespan of this project. So, since "Yes is forever", we are carefully considering new features.

It would be much appreciated if you please provide a bit more context as to why you want this feature? Why is `Note` not enough? What is the use-case? 🙏
